doc/manual/io.rst

```diff
@@ -350,6 +350,14 @@ The `BDF format <http://www.biosemi.com/faq/file_format.htm>`_ is a 24-bit
 variant of the EDF format used by the EEG systems manufactured by a company
 called BioSemi. It can also be read in using :func:`mne.io.read_raw_edf`.
 
+BioSemi amplifiers do not perform "common mode noise rejection" automatically.
+The signals in the EEG file are the voltages between each electrode and CMS
+active electrode, which still contain some CM noise (50 Hz, ADC reference noise,
+etc., see `here <https://www.biosemi.com/faq/cms&drl.htm>`_ for further detail).
+Thus, it is advisable to use any single channel (e.g., Cz) on import of
```

Comment:
This sounds like the user must select a single-channel reference before any other re-referencing operation. That's not how I read the BioSemi FAQ - I think it is important to re-reference the data to data channels (and thereby get rid of the CMS reference), and this includes the average reference.

Reply:
Yes, I like this much better (change it's -> it is).